addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2614,3 +2614,15 @@ we-button-group.o_grid we-selection-items {
         pointer-events: none;
     }
 }
+
+// OUTDATED SNIPPET ALERT
+we-customizeblock-option.snippet-option-VersionControl {
+    position: fixed !important;
```

Comment:
You can use BS classes directly on XML when possible :+1:

Reply:
Yes but I think it is not possible for this one ? 
In web_editor/.../snippets.xml, the option is defined like this 
```xml
<div data-js="VersionControl"
     data-selector="[data-snippet]"/>
```
So I cannot add the class, am I wrong ? :thinking: